ui/app/components/job-versions-stream.js

```diff
@@ -38,7 +40,8 @@ export default class JobVersionsStream extends Component {
         }
       }
 
-      return { version, meta };
+      const diff = this.diffs.objectAt(index);
+      return { version, meta, diff };
```

Comment:
This is the component layer where we opt to join the Version and its Diff, which are otherwise same-indexed objects in separate arrays.